src/main/java/com/ldtteam/structurize/placement/StructurePlacer.java

```diff
@@ -205,7 +205,6 @@ public BlockPlacementResult handleBlockPlacement(
       CompoundTag tileEntityData)
     {
         final BlockState worldState = world.getBlockState(worldPos);
-        boolean sameBlockInWorld = worldState.getBlock() == localState.getBlock();
```

Comment:
the origin of the world equals check was to speed up upgrades where blockstates changed because of some neighbors. Like a fence that now touches something outside of the schematic can be ignored. Maybe we should only bypass this for blockentities then.

Reply:
Yeah, fences connecting outside of schematic is tricky.  Though perhaps we just need a "fences don't care about blockstates" exception?

Reply:
All connecting blocks like fences, walls, panes, etc